src/background.js

```diff
@@ -97,4 +97,18 @@ chrome.runtime.onMessage.addListener((message,sender,sendResponse)=>{
     switch_tab(message.contentPresses);
     console.log("Content Scripts: Switched to " + message.contentPresses + " most recently used tab");
   }
+
+  if (message.name == "CTRL+ArrowLeft") {
```

Comment:
It seems like this will never be called, as we're sending the same message as the CTRL+Q command, and not giving it a name at all.